### DISCUSSION TITLE
UI improvement: Add option to select/deselect all in log viewer drop-down

### DISCUSSION THREAD

**Discussion by jakob-bagterp** [umbraco/Umbraco-CMS#10765] [Features and ideas]
The user can select individual log levels types, but the user can't select/deselect all types in the drop-down. The suggestion is to add an option to select and deselect all log level types.

![Log_Search_-_Settings_-_localhost_-_2021-07-29_11 22 15](https://user-images.githubusercontent.com/25110864/127467040-6fa5b4c9-5a7c-4ca7-a35f-6036ebfa73a7.png)

**Comment by nul800sebastiaan** [comment]
Sure thing, go for it! 👍

**Reply by jakob-bagterp** [reply]
Cool. Pull request created: #10815

**Comment by nathanwoulfe** [comment]
Fixed/implemented and merged in #10815
